Nanoleaf.Client/Nanoleaf.Client/NanoleafStreamingClient.cs

```diff
@@ -0,0 +1,174 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Globalization;
+using System.Linq;
+using System.Net;
+using System.Net.Sockets;
+using System.Threading.Tasks;
+
+namespace Nanoleaf.Client {
+	public class NanoleafStreamingClient : IDisposable {
```

Comment:
I suggest using the same approach for this client as well. The pattern is the same.